spec/models/version_spec.rb

```diff
@@ -151,7 +151,10 @@ def move_to_project(work_package, project)
 
       version.rebuild_positions(project)
 
-      work_packages = version.fixed_issues.find(:all, conditions: { project_id: project }, order: 'COALESCE(position, 0) ASC, id ASC')
+      work_packages = version
+                      .fixed_issues
+                      .where(project_id: project)
+                      .order('COALESCE(position, 0) ASC, id ASC')
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.